07-generic_bookworms/final/collectors/item_test.go

```diff
@@ -0,0 +1,17 @@
+package collectors
+
+type item string
```

Comment:
I would declare them near the test to make the similarity obvious.

Reply:
They were shared in several test files. I'll migrate a type to each test file.

Reply:
Ah, if it's shared then it's fine. For some reason I missed the other scenarios.

Maybe there's a better name for the type? So it's clear what it represents. e.g. `type BookTitle string` or type BookID string`.